opensafely/codelists.py

```diff
@@ -63,39 +68,91 @@ def main():
     pass
 
 
+def add(codelist_url, codelists_dir=None):
+    if not codelists_dir:
+        codelists_dir = Path.cwd() / CODELISTS_DIR
+
+    codelists_file = get_codelist_file(codelists_dir)
+
+    if OPENCODELISTS_BASE_URL not in codelist_url:
+        exit_with_error(f"Unable to parse URL, {OPENCODELISTS_BASE_URL} not found")
+    line = codelist_url.replace(f"{OPENCODELISTS_BASE_URL}/codelist/", "")
```

Comment:
`line` is not a very descriptive name. Is it a codelist descriptor, identifier, name or path, or something else?

Reply:
You're right. I'd copied this convention from the previous code but it's an opportunity to make it better.